rules/eslint/style/walmart.js

```diff
@@ -86,8 +102,8 @@ module.exports = {
     "quote-props": 0,
     // specify whether double or single quotes should be used
     "quotes": [2, "double"],
-    // require identifiers to match the provided regular expression
-    "id-match": 0,
+    // Require JSDoc comment
+    "require-jsdoc": 0,
```

Comment:
This rule is AWESOME and super configurable. I'll probably turn it on for personal projects but I suspect it might get in the way at WM. I'd love to hear thought's from the crowd.

http://eslint.org/docs/rules/require-jsdoc